ui/app/components/clients/token/monthly-new.hbs

```diff
@@ -3,58 +3,51 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="chart-wrapper stacked-charts" data-test-chart="monthly new">
-  <div class="single-chart-grid">
-    <div class="chart-header">
-      <h2 class="chart-title">Monthly new</h2>
-      <p class="chart-description">
-        Entity or non-entity clients which interacted with Vault for the first time during this date range, displayed per
-        month.
+<Clients::ChartContainer
+  @title="Monthly new"
+  @description="Entity or non-entity clients which interacted with Vault for the first time during this date range, displayed per month."
+  @timestamp={{@responseTimestamp}}
+  @hasChartData={{this.hasAverageNewClients}}
+  class={{unless this.hasAverageNewClients "no-legend"}}
+  data-test-chart="monthly new"
+>
+  <:chart>
+    <Clients::VerticalBarChart @dataset={{this.byMonthNewClients}} @chartLegend={{this.runningTotalLegend}} />
+  </:chart>
+
+  <:emptyState>
+    <EmptyState
+      @title="No new clients"
+      @subTitle="There is no new client data available for this {{if
+        @authMount
+        'auth method'
+        'namespace'
+      }} in this date range."
+      @bottomBorder={{true}}
+    />
+  </:emptyState>
+
+  <:stats>
+    <div class="chart-subTitle has-top-padding-l">
+      <h3 class="data-details">Average new entity clients per month</h3>
+      <p class="data-details" data-test-monthly-new-entity>
+        {{format-number this.entityClientData.averageNewClients}}
       </p>
     </div>
```

Comment:
Do you think it's worth making a template only component for this stat display? It looks like the block is the same in all the charts other than the title and number.

Reply:
Yep! I can replace with `StatText`